src/Plugin.php

```diff
@@ -65,12 +65,22 @@ public static function init() {
 
     add_shortcode('user-login-form', __NAMESPACE__ . '\UserLoginForm::getOutput');
 
+    // Disable self pingbacks.
+    add_action('pre_ping', __CLASS__ . '::pre_ping');
```

Comment:
What is a "self pingback"?

If this is about outbound pings, those can be disabled with the first option on `/wp-admin/options-discussion.php` as well as the last text field on `/wp-admin/options-writing.php`.

Reply:
The intention of this code is to avoid getting email notifications when linking some own content in an article. Perhaps it's out of the scope of the task, though. Shall I remove this then?

Reply:
Yes, the goal (for now) is to just get rid of "pings from external sites end up as comments".

Reply:
Do you plan to remove it?